client/coral-embed-stream/src/CommentStream.js

```diff
@@ -108,6 +108,7 @@ class CommentStream extends Component {
               appendItemArray={this.props.appendItemArray}
               updateItem={this.props.updateItem}
               id={rootItemId}
+              premod={this.props.config.moderation}
```

Comment:
I would name this property `moderation` as we may have more than pre/post options in the future.